src/index.js

```diff
@@ -0,0 +1,132 @@
+import './style.css';
+import Draggables from './drag';
+import statusCheck from './status';
+import uiUpdate from './updates';
+
+class TaskList {
+  refereceList = [];
+
+  container = document.querySelector('.container');
+
+  addTaskBtn = document.querySelector('.add-btn');
+
+  listDraggables = new Draggables(this.container);
+
+  constructor(taskList) {
+    this.taskList = taskList;
+
+    this.container.addEventListener('drop', () => {
+      this.save();
+    });
+
+    this.addTaskBtn.addEventListener('click', () => {
+      this.createTodoItem({
+        description: document.querySelector('.add-list').value,
+        completed: false,
```

Comment:
I think you can create this object before and call it whenever you need to create a new item. It will help you keep your code DRY.